experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/OTLPMetricExporterBase.ts

```diff
@@ -117,6 +117,9 @@ export class OTLPMetricExporterBase<
 
   constructor(exporter: T, config?: OTLPMetricExporterOptions) {
     this._otlpExporter = exporter;
+    if (config?.aggregationSelector) {
+     this._otlpExporter.aggregationSelector = config.aggregationSelector
+    }
```

Comment:
This sets the selector to the internal exporter, so It won't be used.
We'd have to do something like [here](https://github.com/open-telemetry/opentelemetry-js/pull/4409/files#diff-0e1a8ae488e7c8045cf2e00ecb2ddb967a7c16d5ccdd089e59a5e79e85e43de5R143-R147) where we implement selectAggregation and then call the selector from the config.

Reply:
[Seems that the necessary line existed already](https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/sdk-metrics/src/export/PeriodicExportingMetricReader.ts#L65)

Reply:
Yep that's in the `PeriodicExportingMetricReader` accessing any `PushMetricExporter` that's passed in (which, in this case is an instance of the class in this file). However this code here sets it on`_otlpExporter` (which is just a field of this class that is used internally), so it's never picked up in the `PeriodicExportingMetricReader`

Reply:
Sorry, I should have had that comment as draft :)